startupscript/butane/idle-shutdown.sh

```diff
@@ -26,6 +26,13 @@ if [[ -z "${IDLE_TIMEOUT_SECONDS}" ]]; then
     exit 0
 fi
 
+UP_TIME="$(awk '{print $1}' /proc/uptime)"
```

Comment:
Worth adding a comment about the issue of old timestamps on boot here.
It is somewhat unrelated to this check here, but I could see a future contributor thinking that this check was unnecessary and removing it; then not discovering it right away as it only crops up on reboot.